swagger-spec/nodes/provider_detail.yaml

```diff
@@ -0,0 +1 @@
+# /nodes/{node_id}/files/providers/{provider}/
```

Comment:
I cannot find any example for this endpoint. I tried https://api.osf.io/v2/nodes/ezcuj/files/providers/osfstorage, but I get response code 502. Could you please find me an example?